Source/WebCore/page/Page.cpp

```diff
@@ -266,14 +266,14 @@ static constexpr OptionSet<ActivityState::Flag> pageInitialActivityState()
     return { ActivityState::IsVisible, ActivityState::IsInWindow };
 }
 
-static Ref<Frame> createMainFrame(Page& page, std::variant<UniqueRef<FrameLoaderClient>, UniqueRef<RemoteFrameClient>>&& client, FrameIdentifier identifier)
+static Ref<Frame> createMainFrame(Page& page, std::variant<UniqueRef<FrameLoaderClient>, UniqueRef<RemoteFrameClient>>&& client, FrameIdentifier identifier, std::optional<ProcessIdentifier> remoteProcessIdentifier)
```

Comment:
This should probably be a variant of UniqueRef<FrameLoaderClient> or a struct RemoteMainFrameCreationParameters that contains a ProcessIdentifier and a UniqueRef<RemoteFrameClient>.  This is because we should never have a RemoteFrameClient but no ProcessIdentifier